pkg/services/pluginsintegration/config/config.go

```diff
@@ -41,15 +26,6 @@ func ProvideConfig(settingProvider setting.Provider, grafanaCfg *setting.Cfg, fe
 		grafanaCfg.PluginsPath,
 		extractPluginSettings(settingProvider),
 		allowedUnsigned,
-		allowedAuth,
-		aws.KeyValue("assume_role_enabled").MustBool(grafanaCfg.AWSAssumeRoleEnabled),
-		aws.KeyValue("external_id").Value(),
-		aws.KeyValue("session_duration").Value(),
-		aws.KeyValue("list_metrics_page_limit").Value(),
-		awsForwardSettingsPlugins,
-		grafanaCfg.Azure,
```

Comment:
what we could do here is to simply pass down `grafanaCfg`? To avoid the insane amount of parameters.